_posts/2019-12-03-ios-learning-group.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: epic
+title: "iOS Learning Group"
+date: "2019-12-03"
+author: [ash]
+categories: [ios, learning, culture]
+---
+
+Regular readers of our blog might be familiar with [Culture Amp](https://www.cultureamp.com), a tool Artsy uses to
+collect anonymous feedback and take action on cultural issues (we most recently discussed the tool
+[in this blog post](https://artsy.github.io/blog/2019/04/19/having-a-coffee-with-every-engineer/)). At a
+company-wide level, Culture Amp has helped guide everything from Artsy's evolving culture, to our physical work
+spaces, to our support for remote work. At an engineering-team level, we've also been using Culture Amp to guide
+our choices in technology, documentation, and training.
+
+In this blog, post, I'll be detailing a recent learning course we ran to share knowledge about how Artsy builds iOS
+software for our entire engineering team.
+
+<!-- more -->
+
+Let's start at the beginning. Earlier this year, Artsy Engineering ran a survey through Culture Amp to get answers
+to the following questions:
+
+1. What is our team's opinion on our current technology choices?
+2. What is our team's familiarity with or preparedness for our current technology?
+3. Where are the areas of strength/opportunities for learning and teaching?
+
+There are a lot of things we learned from this survey, and among them was a desire for engineers to better
+understand how to build iOS software at Artsy. With a nudge (and support!) from our _Peer Learning Working Group_,
+I set out to create a learning plan. I wasn't starting out from scratch – we have already run a few learning
+groups, on topics ranging from Scala fundamentals to React Hooks. We used the lessons learned from _those_
+experiences to define and deliver a learning plan.
+
+I started by booking five sessions, spread out by a week. I picked a time that was a good fit for as many engineers
+as possible, and I scheduled them a month ahead to give people a chance to move their schedules around. Scheduling
+them up front was important, to give learners a sense of what to expect; previous learning groups had suffered from
+inconsistent schedules, which led to intermittent attendance. I also made sure to go to each Tech Lead and ask them
+to make sure at least one engineer from their team attended.
+
+Next was the actual curriculum. In the spirit of "proudly discovered elsewhere", we actually looked at using
+existing learning resources that someone else had already developed. However, since
+[our iOS stack is a bit unique](https://artsy.github.io/series/react-native-at-artsy/) and I already have
+experience developing learning curricula, we decided to make our own. With the help of our Peer Learning Working
```

Comment:
Might wanna leave the 'and I already have experience developing learning curricula,' clause out since it's kinda beside the point and the word 'curricula' is distracting IMO